### DISCUSSION TITLE
Gmail thread reply

### DISCUSSION THREAD

**Discussion by Hritik108** [jeremyephron/simplegmail#104] [Q&A]
I want to implement gmail thread reply, I am able to  get thread id but when I am passing it in send_message it is giving error

**Comment by JTMarsh556** [comment]
Yea, I would like to know how to do this.  It seems like a critical feature since the intent of email is to have threads.  Has anyone figured this out?

**Reply by Hritik108** [reply]
At first, I was trying with SimpleGmail, but later on, I did it using Google's Gmail API.

**Comment by JTMarsh556** [comment]
Yea, I wrote a new function for gmail file to make it work.  I will post it here when I get some time.

**Comment by chrischma** [comment]
This is still an issue. I would love to see your solution. 👍
